packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/TreeSitterScopeHandler/captureUtils.ts

```diff
@@ -15,10 +15,11 @@ export function getRelatedRange(
   scopeTypeType: string,
   relationship: string,
 ) {
-  return getCaptureRangeByName(
-    match,
-    `${scopeTypeType}.${relationship}`,
-    `_.${relationship}`,
+  return (
+    getCaptureRangeByName(match, `${scopeTypeType}.${relationship}`) ??
+    (getCaptureRangeByName(match, scopeTypeType) != null
+      ? getCaptureRangeByName(match, `_.${relationship}`)
+      : undefined)
```

Comment:
this was a bug; if we had `_.iteration`, it would be the iteration scope for every scope, not just the one you wanted 😅.  Thankfully the `#allow-multiple!` requirement surfaced this one, so that's a good argument for requiring that